content/docs/rfcs/41/README.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: 41
+title: 41/WAKU2-RLN-CONTRACT
+name: Rate Limiting Nullifier Contract
+status: raw
+category: RLN
+tags: 17/WAKU-RLN-RELAY
+editor: Keshav Gupta <keshav@status.im>
+contributors:
+---
+
+## Tags
+
+Currently identified tags comprise
+
+* `17/WAKU-RLN-RELAY` protocol is an extension of `11/WAKU-RELAY` which additionally provides spam protection using Rate Limiting Nullifiers (RLN).
+
+
+# Abstract
+The following standard allows for the implementation of a standard API for Rate Limiting Nullifier Contract that manages membership of the participants in a group.
+
+
+# Background / Rationale / Motivation
```

Comment:
In here, we need more details on how each membership is represented, the relation between user sk and pk.

Reply:
Refer 977f0d077f3916ad89231f1020eec7f2846527ed